applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/leds_def.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/* This configuration file is included only once from leds module and holds
+ * information about LED mapping to PWM channels.
+ */
+
+/* This structure enforces the header file is included only once in the build.
+ * Violating this requirement triggers a multiple definition error at link time.
+ */
+const struct {} leds_def_include_once;
+
+/* Mapping the PWM channels to pin numbers */
+static const size_t led_pins[CONFIG_DESKTOP_LED_COUNT]
```

Comment:
LEDs are not used, but I think we can leave it - we will add support for them soon

Reply:
Maybe or maybe not but for sure this won't be using PWM. I will remove this config